rest/model/admin.go

```diff
@@ -759,6 +761,72 @@ func (a *APICloudProviders) ToService() (interface{}, error) {
 	}, nil
 }
 
+type APIContainerPoolsConfig struct {
+	// check APINaiveAuthConfig APIAuthUser to see if its always
+	Pools []*APIContainerPool `json:"pools"`
```

Comment:
Does this need to be a pointer? I'd like to avoid having to handle nil pools in this

Reply:
I don't think so, even though everything in this model seems to use pointers rather than structs themselves -- not exactly sure why. Will change.